pkg/piecestore/psclient/readerwriter.go

```diff
@@ -14,12 +17,24 @@ import (
 	"storj.io/storj/pkg/utils"
 )
 
-// StreamWriter creates a StreamWriter for writing data to the piece store server
+// StreamWriter is astruct for writing data to the piece store server
```

Comment:
typo... also maybe `StreamWriter handles uplink or satellite writing data to piecestore server.`... whether it's "a struct" is really not that important to be mentioned in documentation.

Reply:
done